synapse/handlers/abuse_reports.py

```diff
@@ -205,7 +172,9 @@ async def _report_to_room_moderators(
         event = await self.store.get_event(event_id=event_id, check_room_id=None)
         if event.room_id != room_id.to_string():
```

Comment:
again I ask: why not let `get_event` do this check for you?

Generally it's a pretty poor use of your time and mine for me to make the same review comment multiple times. If something I've written is unclear, just comment on the thread and I'll do my best to clarify.